app/src/js/main.js

```diff
@@ -29,4 +29,37 @@ petDetectrApp.updateDateAndLocationText = function updateDateAndLocationText(e)
     }
 };
 
+// Add listener to nodes
+petDetectrApp.addEventToNodes = function addEventToNodes(evt, nodes, func) {
+    for (let i = 0; i < nodes.length; i++) {
+        nodes[i].addEventListener(evt, func, true);
+    }
+};
+
+// Toggle the h-hide class on target
+petDetectrApp.toggleDisplay = function toggleDisplayOnOff(target) {
+    target.classList.toggle('h-hide');
+};
+
+// Set the use of h-hide class on target
+petDetectrApp.setHideShow = function showHideElement(target, display) {
+    if (display) {
+        target.classList.remove('h-hide');
+    } else {
+        target.classList.add('h-hide');
+    }
+};
+
+// Chip number h-hide switching
+petDetectrApp.hideShowChipNumber = function hideShowChipNumber(e) {
```

Comment:
Yep; I was thinking the same as I was building it.  Some sort of 'on' vs 'off' attribute/hook on the elements.  We just need to decide on the pattern